### PR TITLE
Fix comment typo in standaloneLanguages.ts

### PR DESCRIPTION
**Pull request by hrkw00**
~I just fixed comment typo in `src/vs/monaco.d.ts`.~

Actually , the file is generated automatically. Instead of that, fix a comment in `src/vs/editor/standalone/browser/standaloneLanguages.ts`

I know it's not a big deal, so close it if it's annoying.

### CODE REVIEW

File: src/vs/monaco.d.ts [hunk 1]
```diff
@@ -6241,7 +6241,7 @@ declare namespace monaco.languages {
 
 	/**
 	 * An event emitted when a language is associated for the first time with a text model or
-	 * whena language is encountered during the tokenization of another language.
```
Comment (mjbvz) [inline src/vs/monaco.d.ts:6244]: This file is automatically generated. Instead try finding the original comment in a normal `.ts` file and fix it there

Comment (hrkw00) [reply src/vs/monaco.d.ts]: Thanks @mjbvz !

I found the original file, and fix it in [ddc357c](https://github.com/microsoft/vscode/pull/189449/commits/ddc357ca1cf09bf82c096e6cce63210e5ca3ddf7).

Could you check it again please?

**Review by mjbvz — DISMISSED** [review]

**Comment by mjbvz** [comment]
Thanks but you also need to run the build which will update `monaco.d.ts` based on the changed file

**Comment by hrkw00** [comment]
@microsoft-github-policy-service agree

**Comment by hrkw00** [comment]
@mjbvz I ran `gulp watch` and generate a new 'monaco.d.ts` e892351

Is it ok?

**Review by mjbvz — APPROVED** [review]

**Review by DonJayamanne — APPROVED** [review]

**Pull request merged**